src/main/java/edu/kis/powp/jobs2d/drivers/command/SetPositionCommand.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.jobs2d.drivers.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class SetPositionCommand implements DriverCommand {
+    
+    private Job2dDriver driver;
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).